features/bevragen/persoon/ouder/autorisatie/autorisatie.feature

```diff
@@ -0,0 +1,291 @@
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
+      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
+      | status   | 403                                                                     |
+      | code     | unauthorizedField                                                       |
+      | instance | /haalcentraal/api/brp/personen                                          |
+
+      Voorbeelden:
+      | ad hoc rubrieken              | missende autorisatie                                            |
+      | 10120 10210 10220 10230 10240 | niet geautoriseerd voor oudergegevens                           |
+      | 10120 20210 20220 20230 20240 | niet geautoriseerd voor ouder 2                                 |
+      | 10120 30210 30220 30230 30240 | niet geautoriseerd voor ouder 1                                 |
+      | 10120 28510 28610 38510 38610 | niet geautoriseerd voor oudergegevens in groep 1, 2, 3, 4 of 62 |
+
+    Abstract Scenario: Afnemer is wel geautoriseerd voor <autorisatie>
+      Gegeven de afnemer met indicatie '000008' heeft de volgende 'autorisatie' gegevens
+      | Rubrieknummer ad hoc (35.95.60) | Medium ad hoc (35.95.67) | Datum ingang (35.99.98) |
+      | <ad hoc rubrieken>              | N                        | 20201128                |
+      En de geauthenticeerde consumer heeft de volgende 'claim' gegevens
+      | naam         | waarde |
+      | afnemerID    | 000008 |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000024                       |
+      | fields              | ouders.ouderAanduiding          |
+      Dan heeft de response 1 persoon
+
+      Voorbeelden:
+      | ad hoc rubrieken               | autorisatie                                                  |
+      | 10120 20120 30120              | exact 1 gegeven uit groep 01                                 |
+      | 10120 20240 30240              | exact 1 gegeven uit groep 02                                 |
+      | 10120 20310 30310              | exact 1 gegeven uit groep 03                                 |
+      | 10120 20410 30410              | exact 1 gegeven uit groep 04                                 |
+      | 10120 26210 36210              | exact 1 gegeven uit groep 62                                 |
+      | 10120 20110 30110              | A-nummer uit groep 01 dat niet voorkomt in de resource       |
+      | 10120 20310 30210              | exact 1 gegeven van ouder 1 en 1 ander gegeven van ouder 2   |
+      | 10120 20120 20240 30310 306210 | meerdere gegevens van ouder 1 en andere gegevens van ouder 2 |
+      | PAOU01                         | alleen voorletters van ouder(s)                              |
```

Comment:
moet 10120 hier niet worden toegevoegd? Anders krijgt je de foutmelding dat er niet mag worden gezocht met bsn

Reply:
jazeker. Is al gedaan.